scripts/src/test/bash/integration-test-java-extra

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+CLI="${PWD}/scripts/devon"
+JAVA_VERSION='11*'
+EXTRA_VERSION=$(head -n 1 "${PWD}"/mirrors/java/available-versions)
+
+echo -e "\nJAVA_VERSION=${JAVA_VERSION}" >> "${PWD}/conf/devon.properties"
+echo -e "\nEXTRA_JAVA_VERSION=${EXTRA_VERSION}" >> "${PWD}/conf/devon.properties"
+"${CLI}" java setup
+JAVA_TEST=$(cat "${PWD}"/software/java/.devon.software.version)
+EXTRA_TEST=$(cat "${PWD}"/software/extra/java/.devon.software.version)
+
+[[ $JAVA_TEST == $JAVA_VERSION && $EXTRA_TEST == $EXTRA_VERSION ]]
```

Comment:
I just stumbled over this one:
`$JAVA_TEST == $JAVA_VERSION` will not match as `JAVA_VERSION` will be `11*` but `JAVA_TEST` will be something like `11.0.17_8`.

Reply:
I just fixed it.

Reply:
OK, I was actually wrong here, as I just learned:
```
JAVA_TEST=11.0.17_8
if [[ ${JAVA_TEST} = 11* ]]; then echo match; fi
```
will print "match" :) Sorry, in case of any confusion...